From ebeb735441fd91f2bd80ebda8255f189cb5746ec Mon Sep 17 00:00:00 2001
From: milahu <milahu@gmail.com>
Date: Mon, 18 Mar 2024 15:51:57 +0100
Subject: [PATCH] allow: signup, login, modify account

---
 config.toml | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/config.toml b/config.toml
index d0fc733..7523b4b 100644
--- a/config.toml
+++ b/config.toml
@@ -47,6 +47,15 @@ allowlist = [
     'api-partner\.spotify\.com', # album/artist pages
     'xpui\.app\.spotify\.com', # user interface
     'apresolve\.spotify\.com', # access point resolving
+    'spotify\.com', # signup
+    'www\.spotify\.com', # signup
+    'challenge\.spotify\.com', # signup captcha
+    'challenge\.spotifycdn\.com', # signup captcha
+    'accounts\.spotify\.com', # login
+    'www\.google\.com', # login
+    'accounts\.scdn\.co', # modify account
+    'www\.scdn\.co', # modify account
+    'wap\.spotifycdn\.com', # modify account
     'clienttoken\.spotify\.com', # login
     '.*dealer.?\.spotify\.com', # websocket connections
     'image-upload.*\.spotify\.com', # image uploading
