From b3d0b0911bc3c7daf210fe4290884feb005ba7d3 Mon Sep 17 00:00:00 2001
From: "tsrk." <tsrk@tsrk.me>
Date: Fri, 21 Nov 2025 10:45:37 +0100
Subject: [PATCH] chore: add nix packaging

Signed-off-by: tsrk. <tsrk@tsrk.me>
---
 src/lib.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib.rs b/src/lib.rs
index f804a37..2ab9509 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -46,6 +46,7 @@ lazy_static! {
                 #[allow(deprecated)] // std::env::home_dir() is only broken on Windows
                 Err(_) => PathBuf::from(env::home_dir().unwrap()).join(".config")
             }.join("spotify-adblock/config.toml"),
+            PathBuf::from("@out@/etc/spotify-adblock/config.toml"),
             PathBuf::from("/etc/spotify-adblock/config.toml"),
         ];
 
-- 
2.50.1

